.github/workflows/issue-routing-helper.yml

```diff
@@ -0,0 +1,37 @@
+name: Issue Routing Helper
+on:
+  issues:
+    types: [labeled]
+env:
+  # Use GH_RELEASE_PAT as github-actions bot is not allowed to ping teams
+  GH_TOKEN: ${{ secrets.GH_RELEASE_PAT }}
+  GH_REPO: ${{ github.repository }}
+jobs:
+  route:
+    runs-on: ubuntu-latest
+    if: "startsWith(github.event.label.name, 'Team: ')"
+    steps:
+      - name: "Ensure a single 'Team: *' label with 'Status: Untriaged'"
+        run: |
+          labels_to_remove=$(gh api --paginate "/repos/$GH_REPO/labels" -q '[.[].name | select((startswith("Team: ") or startswith("Status: ")) and . != "${{ github.event.label.name }}" and . != "Status: Untriaged")] | join(",")')
+          gh issue edit ${{ github.event.issue.number }} --remove-label "$labels_to_remove" --add-label '${{ github.event.label.name }},Status: Untriaged'
+      - name: "Mention/ping assigned team for triage"
+        run: |
+          # Get team label mention name:
+          mention_target=$(gh api "/repos/$GH_REPO/labels/${{ github.event.label.name }}" -q '.description')
+          mention_slug=$(gh api "/orgs/getsentry/teams/$mention_target" -q .slug || true)
+          if [[ -z "$mention_slug" ]]; then
+            echo "Couldn't find team mention from slug, trying the label name"
+            team_label='${{ github.event.label.name }}'
+            team_name="${team_label:6}" # Strip the first 6 chars, which is the 'Team: ' part
+            team_name="${team_name// /-}" # Replace spaces with hyphens for url/slug friendliness
+            mention_slug=$(gh api "/orgs/getsentry/teams/$team_name" -q .slug || true)
+          fi
+
+          if [[ -n "$mention_slug" ]]; then
+            message="Routing to @getsentry/$mention_slug for [triage](https://develop.sentry.dev/processing-tickets/#3-triage). ⏲️"
+          else
+            message="[Failed]($GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID) to route to \`${{ github.event.label.name }}\`.  😕\
+                     Defaulting to @getsentry/open-source for [triage](https://develop.sentry.dev/processing-tickets/#3-triage). ⏲️"
```

Comment:
Can we add a couple newlines between these two sentences?

> [Failed](https://github.com/getsentry/release-tester/actions/runs/821326513) to route to `Team: B`.  😕           Defaulting to @getsentry/open-source for [triage](https://develop.sentry.dev/processing-tickets/#3-triage). ⏲️

vs. 

> [Failed](https://github.com/getsentry/release-tester/actions/runs/821326513) to route to `Team: B`.  😕
>
> Defaulting to @getsentry/open-source for [triage](https://develop.sentry.dev/processing-tickets/#3-triage). ⏲️

Reply:
Multiline strings in bash are hard. I thought this added a newline but it didn't.